### ISSUE TITLE
Skrollr on mobile. I doesn't understand

### ISSUE THREAD

**Issue by JLGouwy** [Prinzhorn/skrollr#648]
hi people ! 

In advance, sorry for my bad english. 

this week, I try skrollr for a work project.
Everything works well on desktop. But on mobile, nothing goes.
And I have trouble understanding why.

First, the automatic creation of a hidden overflow (on body) prevents me to scroll.

Second, I don't quite understand the "skrollr-body". Where do i put it ?

Alas, I can not show the project, I suppose it will be difficult to help me.

Basically, here's my structure in picture.

![sturcture](https://cloud.githubusercontent.com/assets/3371103/5281081/8728d714-7af9-11e4-8aef-328e842b44c5.jpg)

sorry for few informations that i give you!

Thanks,
